src/components/ActionBar/src/ActionBarButton.vue

```diff
@@ -183,10 +183,43 @@ export default {
 			assert.error(['squared', 'rounded', 'pill'].includes(shapeValue), `${shapeValue} resolved from ${shapeValueOrPointer} is not a valid shape value and cannot be used in the shape prop of ActionBarButton`);
 			return shapeValue;
 		},
+		resolvedTextColor() {
+			let textColorValueOrPointer;
+			if (this.textColor) {
+				textColorValueOrPointer = this.textColor;
+			} else {
+				textColorValueOrPointer = this.theme.actionbarbutton.textColor;
+			}
```

Comment:
why not:
`const textColorValueOrPointer = this.textcolor || this.theme.actionbarbutton.textColor;`

Reply:
i'll make a follow-up refactor PR to clean up the repetitive computed methods used to resolve themeable props.